src/builtin-templates/flash-error.js

```diff
@@ -1,20 +1,25 @@
-export default function(element, err) {
-    console.error(err);
+import { html, render } from '/web_modules/lit-html/lit-html.js';
+import { unsafeHTML } from '/web_modules/lit-html/directives/unsafe-html.js';
 
-    if (!element) {
-        return;
+export default function create(error) {
+    const el = document.querySelector('#flash-error');
+
+    if (error) {
+        console.error(error);
     }
 
-    const message = (typeof err === 'string' ? err : 'Please check the items in red below.')
-        .replace(/&/g, '&amp;')
-        .replace(/</g, '&lt;')
-        .replace(/>/g, '&gt;');
+    return {
+        show: () => { render(wrap(error), el); },
+        hide: () => { render('', el); }
+    };
+}
+
+function wrap(err) {
+    const message = typeof err === 'string' ? err : 'Please check the items in red below.';
 
-    return element.innerHTML = `
+    return html`
         <div class="flash-error">
             <p><b class="large">We’re sorry, something is missing or wrong.</b></p>
-            <p>${message}</p>
-        </div>
-    `;
+            <p>${ unsafeHTML(message) }</p>
```

Comment:
I'm not sure it's wise to put an unescaped error into HTML.

Reply:
I propose to change this and to remove dynamic error message display
because of this https://ubio-automation.atlassian.net/browse/SDK-74
and also because it never was an explicit requirement, to display any part of the real error. Let's just keep `console.error` for developer's benefit